dbbuild/sources/1kgp/mhfreqs.py

```diff
@@ -46,6 +46,9 @@ def construct_haplotypes(rsids, vcf, rsidxfile):
                     samples = line.strip().split()[9:]
                 continue
             fields = line.strip().split()
+            rsid = fields[2]
+            if rsid not in rsids:
+                continue
```

Comment:
One of the markers is spanned by two CNVs, resulting in bookkeeping errors when aggregating the marker SNP data. These change to the build code eliminates that error.